PostBuild.sh

```diff
@@ -250,12 +250,11 @@ function GrubSetup {
       err_exit "Bare partitioning not yet supported"
    else
       ROOTTOK="root=${VGCHECK}"
+      VGCHECK="${VGCHECK%-*}"
 
       # Compute PV from VG info
       CHROOTDEV="$(
-            vgs --no-headings -o pv_name "$( 
-               sed "s#/dev/mapper/##" <<< "${VGCHECK%-*}"
-            )" | \
+            vgs --no-headings -o pv_name "${VGCHECK//\/dev\/mapper\//}" | \
```

Comment:
Nice. Could swear that, when I was working on it, I'd had to resort to the `sed` kludge because it wasn't allowing me to escape the `/`s.